templates/web/northamptonshire/footer_extra.html

```diff
@@ -1,5 +1,5 @@
 <footer class="northamptonshire">
-    <p>© Northamptonshire County Council 2019</p>
+    <p>© Northamptonshire County Council 2021</p>
```

Comment:
How do we feel about replacing this with a programatically `current year`?

Reply:
Good point, will do it as a separate commit as I'd imagine there's a few cobrands with copyright in footer.